QA fixes (4/30) (#1314)
Kong/developer.konghq.com: app/_kong_plugins/prometheus/index.md

categories:
  - analytics-monitoring

faqs:
  - q: If I have multiple Data Plane nodes with the Prometheus plugin enabled, does the metrics data reflect the status of a single node or the aggregated status across all nodes?

>     a: The data are reported per-node (each Data Plane) per-subsystem (HTTP or TCP).

Reviewer: ```suggestion
    a: Metrics data is reported per-node (each Data Plane) per-subsystem (HTTP or TCP).
```
